dbt/models/reporting/schema.yml

```diff
@@ -111,6 +111,59 @@ models:
           name: reporting_vw_pin_most_recent_sale_not_null
           expression: sale_price IS NOT NULL
 
+  - name: reporting.vw_pin_township_class
+    description: '{{ doc("view_vw_pin_township_class") }}'
+    tests:
+      - not_accepted_values:
+          name: reporting_vw_pin_township_class_class_no_hyphens
+          column_name: class
+          values: "2-99"
+      - unique_combination_of_columns:
+          name: reporting_vw_pin_township_class_unique_by_14_digit_pin_year_stage
+          combination_of_columns:
+            - pin
+            - year
+      - not_null:
+          name: reporting_vw_pin_township_class_class_not_null
+          column_name: class
+          config:
+            where: CAST(year AS int) < {{ var('test_qc_year_start') }}
```

Comment:
**question:** What's this intended to do? Why test everything _before_ the more recent years?

Reply:
There was a point to this at some point, I think there were new PINs for 2024 with NULL classes, but that doesn't seem to be the case any longer and this config isn't needed.